Makefile

```diff
@@ -30,7 +30,11 @@ install: deps
 
 # Lint markdown files
 lint:
+ifeq ($(LINTER),)
+	$(error "Linter is not installed, run make deps to install it")
```

Comment:
this was so much needed! i would also suggest updating the contribution guidelines to first run `make deps` and then `make lint`